santosfamilyfoundation/RuinaMain
# Sherrie county state dropdown integration

manual test: 
1) select state, select county, then select state again, county answer should be cleared and status bar turns red
2) without selecting state, select county, if choose please select a state..., county answer should remain unpopulated and status bar remains red
